version.json

```diff
@@ -0,0 +1,6 @@
+{
+    "build": "TBD",
```

Comment:
Happy to automate what these values should be. Not sure what "build" is, though.

Reply:
Circle will fill in the correct values here: https://github.com/mozilla-services/pushbox/blob/master/.circleci/config.yml#L25

Reply:
cool, thanks!